pallets/pips/src/lib.rs

```diff
@@ -1324,8 +1325,9 @@ impl<T: Config> Module<T> {
     /// Panics if the PIP doesn't exist or isn't scheduled.
```

Comment:
This function doc is wrong.  The function doesn't "Panic", it returns an error if the PIP doesn't exist or is not scheduled.  Please rewrite this comment.  Thanks.